app/src/main/res/layout-land/profile_progress_header.xml

```diff
@@ -10,174 +10,172 @@
-          android:layout_gravity="end|bottom"
-          android:onClick="@{() -> viewModel.clickOnProfilePicture()}"
-          android:src="@drawable/ic_edit_18"
-          app:backgroundTint="@color/white"
-          app:fabSize="normal"
-          app:tint="@color/colorPrimary" />
-      </FrameLayout>
+      android:layout_marginTop="12dp"
+      android:fontFamily="sans-serif-medium"
+      android:text="@{viewModel.profile.name}"
+      android:textColor="@color/oppiaPrimaryText"
+      android:textSize="28sp"
+      app:layout_constraintStart_toEndOf="@+id/profile_edit_image"
+      app:layout_constraintStart_toStartOf="@+id/ongoing_topics_container"
+      app:layout_constraintTop_toTopOf="parent" />
+
+    <com.google.android.material.card.MaterialCardView
+      android:id="@+id/ongoing_topics_container"
+      android:layout_width="0dp"
+      android:layout_height="wrap_content"
+      android:layout_marginStart="44dp"
+      android:layout_marginTop="20dp"
+      android:layout_marginEnd="20dp"
+      android:layout_weight="1"
+      android:clickable="@{viewModel.ongoingTopicCount > 0}"
+      android:onClick="@{() -> viewModel.clickOnOngoingTopicCount()}"
+      app:cardElevation="4dp"
+      app:contentPadding="8dp"
+      app:layout_constraintEnd_toStartOf="@+id/completed_stories_container"
+      app:layout_constraintHorizontal_weight="1"
+      app:layout_constraintStart_toEndOf="@+id/profile_edit_image"
+      app:layout_constraintTop_toBottomOf="@+id/profile_name_text_view">
 
       <LinearLayout
-        android:layout_width="wrap_content"
+        android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:layout_marginStart="32dp"
-        android:orientation="vertical">
+        android:layout_gravity="center"
+        android:gravity="center_horizontal"
+        android:orientation="vertical"
+        android:padding="8dp">
 
         <TextView
-          android:id="@+id/profile_name_text_view"
+          android:id="@+id/ongoing_topics_count"
           android:layout_width="wrap_content"
           android:layout_height="wrap_content"
-          android:layout_marginTop="12dp"
-          android:fontFamily="sans-serif-medium"
-          android:text="@{viewModel.profile.name}"
+          android:fontFamily="sans-serif"
+          android:text="@{Integer.toString(viewModel.ongoingTopicCount)}"
           android:textColor="@color/oppiaPrimaryText"
-          android:textSize="28sp"
-          app:layout_constraintEnd_toEndOf="parent"
-          app:layout_constraintStart_toStartOf="parent"
-          app:layout_constraintTop_toBottomOf="@+id/profile_frame_layout" />
+          android:textSize="24sp" />
 
-        <LinearLayout
-          android:id="@+id/progress_stats_linear_layout"
-          android:layout_width="wrap_content"
+        <TextView
+          android:id="@+id/ongoing_topics_description_text_view"
+          android:layout_width="60dp"
```

Comment:
Are we supposed to give fixed width here.

Reply:
As per the mock and also its having that text are limit.

Reply:
Done.